pygmt/__init__.py

```diff
@@ -18,7 +18,7 @@
 from .gridding import surface
 from .sampling import grdtrack
 from .mathops import makecpt
-from .modules import config, info, grdinfo, which
+from .modules import config, info, grdinfo, grdcut, which
```

Comment:
Probably best to start a separate file instead of putting it in `modules.py`. GMT puts grdcut under ["Convert or extract subsets of data"](https://docs.generic-mapping-tools.org/latest/modules.html#convert-or-extract-subsets-of-data). Can't come up with a good name, but maybe `conversion.py`, or `extraction.py`, or `subset.py`?

Reply:
As it's invisible to users, perhaps we can do the moving after we have more functions in `modules.py` and have a better idea to organize them?

Reply:
Yes, but it's harder to run tools like `git blame` on files that have moved unless people are very good with `git`. Best to put it in the right place from the start.

Reply:
Fair enough. We already have a `clib/conversion.py` file, so `conversion.py` is not a good name.

Reply:
Sigh, I wished we had just put everything as a separate `*.py` file like in [GMT.jl](https://github.com/GenericMappingTools/GMT.jl/tree/master/src) instead of grouping them :facepalm:. Would `converter.py` do? Or `convertsubset.py`? I'm really bad with names :cry:

Reply:
> I wished we had just put everything as a separate `*.py` file like in [GMT.jl](https://github.com/GenericMappingTools/GMT.jl/tree/master/src) instead of grouping them

Me too. It's also easier to find the functions.

Reply:
It seems all modules were first designed to be in a `modules` directory (https://github.com/GenericMappingTools/design-python#package-organization), but later Leo changed his mind (https://github.com/GenericMappingTools/pygmt/commit/252d5df5f493ca2dafb3bed2a0b6909e37c40578)

Reply:
What about [`gridops.py`](https://docs.generic-mapping-tools.org/latest/modules.html#grid-operations), similar to the `mathops.py` we already have. I think it makes sense to have grdcut, grdpaste and grdblend in that group. 

BTW, I don't think it's a good choice to put modules by groups. We probably need to open a new issue discussing if we want to put modules in separate *.py files.

Reply:
Yeah, let's do `gridops.py` for now. I don't think we can avoid renaming/shifting the modules in the future, but that's a topic for another issue.